Tests/HederaE2ETests/Topic/TopicMessageSubmit.swift

```diff
@@ -134,4 +134,62 @@ internal class TopicMessageSubmit: XCTestCase {
 
         _ = try XCTUnwrap(transaction.transactionId)
     }
+
+    internal func testSubmitMessage() async throws {
+        let testEnv = try TestEnvironment.nonFree
+
+        let topic = try await Topic.create(testEnv)
+
+        addTeardownBlock {
+            try await topic.delete(testEnv)
+        }
+
+        let key = PrivateKey.generateEd25519()
+
+        let receipt = try await AccountCreateTransaction(key: .single(key.publicKey), initialBalance: 50)
+            .execute(testEnv.client)
+            .getReceipt(testEnv.client)
+
+        let id = try XCTUnwrap(receipt.accountId)
+
+        do {
+            // Set user account
+            let userAccountId = id
+            let userPrivateKey = key
+            let userClient = Client.forTestnet().setOperator(userAccountId, userPrivateKey)
+
+            // Set payer account
+            let payerAccountId = testEnv.operator.accountId
+            let payerPrivateKey = testEnv.operator.privateKey
+            let payerClient = Client.forTestnet().setOperator(payerAccountId, payerPrivateKey)
```

Comment:
This (and the other client) makes a testnet client but the test env could be not testnet

Reply:
Additionally, this should probably just flat out be `testEnv.client`